example/lib/main.dart

```diff
@@ -74,7 +74,7 @@ class _MyAppState extends State<MyApp> {
   }
 
   _fetchMicrosoftProfile() async {
-    var response = await http.get(this._graphURI, headers: {"Authorization": "Bearer " + this._authToken});
+    var response = await http.get(this._graphURI as Uri, headers: {"Authorization": "Bearer " + this._authToken});
```

Comment:
Is downcasting alright? maybe we should Parse the url as Uri

Reply:
It is an example project, that is alright